Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM cgr.dev/chainguard/wolfi-base
-LABEL org.opencontainers.image.source https://github.com/github.com/uptime-induestries/compute-blade-agent
+LABEL org.opencontainers.image.source="https://github.com/github.com/uptime-industries/compute-blade-agent"
 
 # Copy binaries generated by goreleaser
-COPY compute-blade-agent bladectl /bin
+COPY compute-blade-agent bladectl /bin/
```

Comment:
nice catch! Haven't used the image yet

Reply:
Me neither, but all hail to the almighty linter in my IDE 😆